Puppetfile

```diff
@@ -1,15 +1,14 @@
 forge "http://forge.puppetlabs.com"
 
-mod 'epel',                  :git => 'git://github.com/stahnma/puppet-module-epel.git',                         :ref => '1.1.1'
-mod 'stdlib',                :git => 'git://github.com/puppetlabs/puppetlabs-stdlib.git',                       :ref => '4.6.0'
-mod 'inifile',               :git => 'git://github.com/puppetlabs/puppetlabs-inifile.git',                      :ref => '3863d9a9c1e314be73f3517a481f0ad1f2f7a772' # 1.4.3
+mod 'epel',                  :git => 'git://github.com/stahnma/puppet-module-epel.git',                         :ref => '1.3.0'
+mod 'stdlib',                :git => 'git://github.com/puppetlabs/puppetlabs-stdlib.git',                       :ref => '4.24.0'
+mod 'inifile',               :git => 'git://github.com/puppetlabs/puppetlabs-inifile.git',                      :ref => '2.1.1'
 mod 'firewall',              :git => 'git://github.com/puppetlabs/puppetlabs-firewall.git',                     :ref => '1.7.1'
-mod 'java',                  :git => 'git://github.com/danielparks/puppetlabs-java.git',                        :ref => '07d2d9d79f26922ef5db2115b4f036c8c628a019'
-mod 'jenkins',               :git => 'git://github.com/jenkinsci/puppet-jenkins.git',                           :ref => 'd70fd6f6ccc5a8eae7a9f7c1f7d17efd34ad87e7'
-mod 'archive',               :git => 'git://github.com/voxpupuli/puppet-archive.git',                                     :ref => 'v0.4.8'
-mod 'python',                :git => 'git://github.com/stankevich/puppet-python.git',                           :ref => 'e7389f6e270796372b93b178fd5295abefbdc17a' # 1.10.0
-mod 'rvm',                   :git => 'git://github.com/waynr/puppet-rvm.git',                                   :ref => '53c564e56ff8a9b284dfa0bd03e9b9523cc8c3bb'
-# soon-ish, we should pin this to a tag or a SHA, but I expect a little more churn for a while,
-#  so pointing it to `master` for now.
```

Comment:
This comment is for the line we keep below (puppetserver_perf_driver), right? Should we leave the comment or tag the repo?

Reply:
Either way is fine with me. I didn't know if we ever had any intent to tag it, so I removed the comment because it seems somewhat untrue. But if we do want to tag it, that works for me.

Reply:
My reading of the comment is that they intended, once they weren't iterating rapidly on it, to tag the repo, point to the tag here, and remove the comment. I think that's a good thing and we should go ahead and do that once https://github.com/puppetlabs/puppetlabs-puppetserver_perf_driver/pull/19 is merged.

Reply:
Sounds good, we can wait to merge this till we have that tag then.

Reply:
1.4.0 is now tagged

Reply:
Updated.